configs/mot/deepsort/README.md

```diff
@@ -1,8 +1,18 @@
-# Deep SORT
+# Simple online and realtime tracking with a deep association metric
```

Comment:
please delete this [line](https://github.com/open-mmlab/mmtracking/blob/master/model-index.yml#L9)